Cosmic cult entropy balance tweaks

NoElkaTheGod opened the pull request:
## About the PR
1. Cosmic cult now needs slightly more entropy to progress (7 -> 10 entropy per convert)
2. Siphoning entropy now deals it's damage slower.
3. Siphoning entropy no longer gets interrupted when changing hands or dropping an item.

## Why / Balance
1. Entropy was worth far too much for progression. Why bother getting yourself force ingress, if you can convert 2 engineers for the same amount of progress, who will probably be even better at opening doors?
2. The effect was far too obvious, which led to very awkward interactions ("Urist McCultist just stood next to me and now I'm suffocating? Hmmmm, yeah, that's probably an atmos issue despite me not gasping or anything") and sometimes borderline metagaming. I did not want to outright remove it, so I left it for the empowered cultists, because siphoning while empowered is fairly obvious already, thansk to the lights blinking.
3. Why the fuck it did in the first place? Not like you're using an item in your hand to do it.

## Technical details
Not really.

## Media
Not really.

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- tweak: Entropy is now less efficient for progressing the cosmic cult's monument.
- tweak: Siphoning entropy now deals it's damage much slower, and the total amount is randomized to make it less metagameable.

NoElkaTheGod committed:
Decrease entropy value

NoElkaTheGod committed:
Adjust siphoning

NoElkaTheGod commented:
Some more on entropic degen. It was initially intended to provide an eventual disruption, so that if you keep siphoning from one person, they'll eventually have to go to medbay. However, degen deals enough asphyxiation damage to change the target's health indicator, alerting them that something is wrong, which would then usually lead to them finding an excuse to move somewhere else.
Additionaly, the actual challenging part of siphonin entropy isn't even the damage management (you literally only need to use 1 ointment per siphon, or have a medibot), it's catching a moment when someone stands still, and having an actual reason to approach them as to not look suspicious.

NoElkaTheGod commented:
Test fail unrelated

Proxysseia commented:
23h, as a treat

Proxysseia commented on Content.Server/_DV/CosmicCult/Abilities/CosmicSiphonSystem.cs:
```diff
@@ -96,6 +96,7 @@ private void OnCosmicSiphonDoAfter(Entity<CosmicCultComponent> uid, ref EventCos
 
         if (uid.Comp.CosmicEmpowered) // if you're empowered there's a 50% chance to flicker lights on siphon
         {
+            _statusEffects.TryAddStatusEffect<CosmicEntropyDebuffComponent>(target, "EntropicDegen", TimeSpan.FromSeconds(21), true); //only applies when empowered now
```
we want some kind of harm from siphoning, be that completely random damage amounts or types, be it delayed by a minute, something.

Proxysseia commented:
rest is fine

NoElkaTheGod commented:
Noted. I will try to cook something later, then.

NoElkaTheGod committed:
Change entropyDegen

NoElkaTheGod commented:
Changed the debuff applied when siphoning to apply it's damage slower (0.1 cold per second), and made the duration random (40-60 seconds) to make it less metagameable. No longer deals asphyxiation, since that healed by itself anyway (and didn't affect IPCs because of course it didn't).

Also removed some unused dependencies.

NoElkaTheGod commented:
Wait hold on, it uses the same effect for the cosmicEquipment, and I don't wanan nerf that (it's kinda weak already)

NoElkaTheGod committed:
Un-nerf cosmic equipment debuff

NoElkaTheGod commented:
Alright, I think that's all the feedback adressed

NoElkaTheGod commented:
@deltanedas any problems here?

deltanedas commented on Content.Server/_DV/CosmicCult/CosmicCultSystem.cs:
```diff
@@ -166,7 +166,10 @@ private void OnGetVisMask(Entity<CosmicCultComponent> uid, ref GetVisMaskEvent a
+            {
+                _statusEffects.TryAddStatusEffect<CosmicEntropyDebuffComponent>(args.Equipee, EntropicDegen, TimeSpan.FromDays(1), true); // TimeSpan.MaxValue causes a crash here, so we use FromDays(1) instead.
+                if (TryComp<CosmicEntropyDebuffComponent>(args.Equipee, out var comp)) comp.Degen = new(){DamageDict = new(){{"Cold", 0.5}, {"Asphyxiation", 1.5}}};
+            }
```
this is overindented

NoElkaTheGod committed:
Unindent by 2 spaces ops

NoElkaTheGod commented:
Tomorrow's changelog entry will be quite a lot of cult tweaks

NoElkaTheGod commented:
The nefarious unresolved comment
